drivers/interrupt_controller/Kconfig.multilevel

```diff
@@ -52,6 +52,33 @@ config NUM_2ND_LEVEL_AGGREGATORS
 	  aggregator can manage at most MAX_IRQ_PER_AGGREGATOR level 2
 	  interrupts.
 
+config 1ST_LEVEL_INTERRUPT_BITS
+	int "Total number of first level interrupt bits"
+	range 1 32
+	depends on MULTI_LEVEL_INTERRUPTS
+	help
+	  The number of bits to use of the 32 bit interrupt mask for first
+	  tier interrupts this interpretation forgoes the idea of aggregators
+	  and simply uses the bit vector to generate irqs.
+
+config 2ND_LEVEL_INTERRUPT_BITS
+	int "Total number of second level interrupt bits"
+	range 1 31
+	depends on 2ND_LEVEL_INTERRUPTS
+	help
+	  The number of bits to use of the 32 bit interrupt mask for second
+	  tier interrupts this interpretation forgoes the idea of aggregators
+	  and simply uses the bit vector to generate irqs.
+
+config 3RD_LEVEL_INTERRUPT_BITS
```

Comment:
I think it would make sense to move this down to where the other `3RD_LEVEL_` options are.